Twilio-org/phonebank
# Place call with Twilio

Description:
- Add callVolunteer method to the Twilio module
- Rename updateUserCallSIDField controller name to startTwilioConnection
- Update startTwilioConnection controller to call a volunteer and save the call_sid to the users table
- Comment out authentication on user routes to allow callbacks to happen

Tests:
- Creating PR without automatic tests in order to unblock other tickets. Creating a new ticket for testing (Jira 147)
- Manual Test:
  - Register a user with your phone number
  - Login as the user and join Campaign 1 from seed file
  - Query database to confirm new user_id
  - Run the following curl command:
```
curl -X POST \
  http://localhost:3000/users/<user_id>/campaigns/1/calls/ \
  -H 'cache-control: no-cache' \
  -H 'postman-token: 5ce03639-4e27-ac5a-fe78-6ef21f481f33'
```
- You should receive a call the the phone number your registered and the command should return the call_sid and updated_at timestamp.

Jira Ticket 122 & 132
